Dockerfiles/manifests/openshift/tf-cluster/main.tf

```diff
@@ -0,0 +1,28 @@
+//  Setup the core provider information.
+provider "aws" {
+  region  = "${var.region}"
```

Comment:
make sure we use a specific version of the `aws` plugin

Reply:
added version